pages/Transaction.tsx

```diff
@@ -138,8 +160,19 @@ export function Transaction() {
             />
             <TransactionDetailRow
               title="Description"
-              content={transaction.description || "-"}
+              content={
+                // TODO: should this description be merged on Alby Hub side?
```

Comment:
I think it is best to keep the description as BOLT11 invoice description and use a separate title and text for metadata related fields.

But I think `transaction.description || metadata?.recipient_data?.description || metadata.comment` can be used for the tx list to give more context than before.